lib/ueberauth/strategy/amco/user.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Ueberauth.Strategy.Amco.User do
```

Comment:
This struct will be assigned in `conn.assigns[:current_user]` in the AuthenticatedUser plug.